modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -35,5 +35,13 @@
 		],
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
+	},
+	{
+		"from": "CaptchaUtil.serveImage(ResourceRequest, ResourceResponse)",
+		"issueKey": "LPS-196215",
+		"newImports": [
+			"com.liferay.portal.kernel.util.PortalUtil"
+		],
+		"to": "CaptchaUtil.serveImage(PortalUtil.getHttpServletRequest(param#0#), PortalUtil.getHttpServletResponse(param#1#))"
```

Comment:
@TamyrisTorres can we move the `CaptchaUtil` to the classNames attribute and avoid this repetition on the `from` and `to` attributes? Ideally, we should only add the class to the `from` and `to` attributes when the class needs to be replaced as well.